cmake/ElektraCompiling.cmake

```diff
@@ -84,9 +84,9 @@ set (COMMON_FLAGS "${COMMON_FLAGS} -Wsign-compare -Wfloat-equal")
 set (COMMON_FLAGS "${COMMON_FLAGS} -Wformat-security")
 set (COMMON_FLAGS "${COMMON_FLAGS} -Wshadow")
 set (COMMON_FLAGS "${COMMON_FLAGS} -Wcomments -Wtrigraphs -Wundef")
-set (COMMON_FLAGS "${COMMON_FLAGS} -Wuninitialized -Winit-self -Wmaybe-uninitialized")
```

Comment:
Is Wmaybe-uninitialized unknown to the apple compiler? Ideally we should have a ```if (apple)``` and move non-common flags to the specific places.